OKR/CLUSTER.md

```diff
@@ -46,6 +46,7 @@ The following may not be for next quarters as realistically they will very likel
 ### O4. Cluster outreach
 
 * KR: Finish illustrations - P0 (Mhz)
+* KR: Cluster Starter App - P1 (pkafei)
```

Comment:
what is Cluster Starter App @pkafei ?

Reply:
The Cluster Starter app is connected to [the Truffle Box task](https://docs.google.com/document/d/1Pk_XjQbzcFtaEWWhhkfMyX8OLZPSTGOH6eutRB3z1O8/edit). The Cluster Starter App will provide boiler plate code for the user to create an application.

Reply:
We should discuss this further. I'm not in favor of having official docs anywhere other than the cluster website. We can get good ideas about what to do with our documentation from this.